OakCityV2/TableViewController.m

```diff
@@ -0,0 +1,190 @@
+//
+//  TableViewController.m
+//  OakCityV2
+//
+//  Created by Josh Green on 6/8/15.
+//  Copyright (c) 2015 Josh Green. All rights reserved.
+//
+
+#import "TableViewController.h"
+#import "AFNetworking/AFNetworking.h"
+#import "DetailViewController.h"
+#import "DetailCell.h"
+#import "Job.h"
+#import "AFNetworking/UIImageView+AFNetworking.h"
+#define LANGUAGE @"PHP"
+#define TOWN @"San+Francisco"
+
+@interface TableViewController ()
+
+@end
```

Comment:
You've got a property in the corresponding header file called: 

@property (strong, nonatomic) NSArray *jobs;

Since it's in the header file, it's public.  But you don't really want it to be public.  The fact you have a jobs list and use it to keep track of things for display purposes is an implementation detail that doesn't need to be exposed to the world.  So you should move it to the TableViewController extension here so that it's private.